core/modules/node/node.module

```diff
@@ -2189,7 +2189,11 @@ function node_feed($nids = FALSE, $channel = array()) {
       ->fetchCol();
   }
 
-  $item_length = $rss_config->get('rss_viewmode');
+  $display_mode = $rss_config->get('rss_viewmode');
+  if ($display_mode == 'fulltext') {
```

Comment:
Why do we need to change from `fulltext` to `full`, was this one of the setting values in Drupal 7?  I'll go check...

Reply:
Ah, I see. It was the default value for `feed_item_length`... 
```
$item_length = variable_get('feed_item_length', 'fulltext');
```

Reply:
@jenlampton 

I thought about changing the setting from "fulltext" to "full", but that will break things for people upgrading from previous Backdrop versions or migrating from D7. 

That can be avoided by having an update hook to change it, but at this point, it is more work for something that may eventually be all replaced with views anyway (see discussion in #1583 for more)

Another point: the automated tests keep failing, and I have no idea why ...